docs/source/resources/01a_object_detection.rst

```diff
@@ -80,7 +80,7 @@ The following test conditions were followed:
      frames to process per video
  | - 1280×720 (HD ready) resolution was used, as a bridge between 640×480 (VGA) of poorer quality
      webcams, and 1920×1080 (Full HD) of CCTVs
- | - All unnecessary processes, such as browsers, were closed to prevent IO/resource contention
+ | - All tests were performed in a containerised environment running on Google Cloud VMs
```

Comment:
We have decided to go with US English so it works better with external libraries. Please update this to use US Eng as well.